app/src/main/res/values/strings.xml

```diff
@@ -103,6 +103,8 @@
 
     <!-- Pinned Messages -->
     <string name="title_pinned_messages">Pinned Messages</string>
+    <string name="no_pinned_messages">No pinned Messages</string>
+    <string name="no_pinned_description">All the pinned messages   appear here.</string>
```

Comment:
Could you add the Brazilian translation for this string?

All the pinned messages appear here -> Todas as mensagens pinadas aparecerão aqui.